src/main/java/pl/edu/agh/ki/io/models/wallElements/Photo.java

```diff
@@ -0,0 +1,14 @@
+package pl.edu.agh.ki.io.models.wallElements;
+
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Getter
+@NoArgsConstructor
+@Entity
+@Table(name = "photos")
+public class Photo extends WallItem {
```

Comment:
To photo to są zdjęcia profilowe i posty ze zdjęciami userów, ale już nie zdjęcia szczytów?

Reply:
Myslalem zeby to byly zdjecia uzytkownikow ale nie zdjecia profilowe, bo wtedy trzeba by bylo jakos rozrozniac w bazie ktore zdjecie jest profilowe a ktore 'zwykle', zdjecie profilowe jest przechowywane w userze, tak samo zdjecia szczytow
Nie wiedzialem jak to rozwiazac w sumie, wiec jak masz jakis inny pomysl to chetnie przyjme  
 teraz mi jeszcze przyszlo do glowy ze zdjecie tez moze miec opis wiec by sie przydalo to dodac

Reply:
I zdjęcie profilowe jest mimo wszystko także zdjęciem, które znajdzie się na wallu. W takim razie wydaje mi się że jednym z rozwiązań może być idea, że każde zdjęcie usera to wall Item i akurat zdjęcie profilowe jest wyróżnione

Reply:
dodam flage w takim razie czy zdjecie jest normalne czy profilowe

Reply:
Ewentualnie można pomyśleć w inny sposób. Mamy posty, które są wall Itemami. I teraz w poście może znajdować się zdjęcie, może znajdować się trasa lub może znajdować się tekst. Baza nie musi wtedy wiedzieć że dane zdjęcie jest akurat na wallu, my obsługujemy to w kontrollerze czy tam w repository